packages/browser/src/plugins/destinations/__tests__/index.test.ts

```diff
@@ -32,16 +32,9 @@ describe('destination', () => {
       next(payload)
     })
 
-    const htevents = HtEventsBrowser.load(
-      {
-        writeKey: 'WRITE_KEY',
-      },
-      {
-        destinations: {
-          Test: destination,
-        },
-      }
-    )
+    const htevents = HtEventsBrowser.load({ writeKey: 'WRITE_KEY' })
+
+    await htevents.register(destination)
```

Comment:
Why did we have to change this code? Does the passing of destinations into the load function still work in a non-testing scenario?

Reply:
I simplified things a little bit here by making the `InitOptions.destinations` object strictly for passing settings to pre-implemented destinations. If customers want to add a custom `Destination` plugin they can do so using the existing `htevents.register` function (which is already documented). Figured it was simpler to just have a single path for registering custom plugins/destinations via `htevents.register`.